lua/cmp_fuzzy_path/init.lua

```diff
@@ -83,7 +81,7 @@ local function find_pattern(cursor_before_line)
 end
 
 source.complete = function(self, params, callback)
-  params.option = vim.tbl_deep_extend('keep', params.option, defaults)
+  opts = vim.tbl_deep_extend('keep', params.option, opts)
```

Comment:
The correct pattern is to extend the `params.option`, not your global `opts`, as there may be several entry points into the plugin from cmp.
Please change.